rootfs/templates/kops/kubecfg.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: Config
+preferences: {}
+
+clusters:
+- cluster:
+    server: https://{{ getenv "KOPS_API_SERVER" }}
```

Comment:
is there a `requiredEnv` type function we can use?

Reply:
Unfortunately not with gomplate https://gomplate.hairyhenderson.ca/functions/env/ but we assert it has been set in the script that builds the kubecfg

Reply:
`requiredEnv` is helmfile specific